### PR TITLE
Extend conda testing

### PR DESCRIPTION
**Pull request by JanuszL**
- changes conda build version from the git sha to version+timestamp for regular builds
- adds more conda test variants
- adds build number to conda build
- makes dali use custom opencv (without threads, cuda, with libjpeg-turbo) and ffmpeg builds (with mpeg4_unpack_bframes) as the Conda provided, (with all features enabled) doesn't work well with mpeg4_unpack_bframes
- makes all gtest and python basic tests pass
- sets libvorbis version to 1.3.5 due to https://github.com/conda-forge/libvorbis-feedstock/issues/14

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It adds more test variants for conda build and improves DALI conda build process

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     changes conda build version from the git sha to version+timestamp for regular builds
     adds build number to conda build
     adds more conda test variants
     makes dali use custom opencv and ffmpeg builds
 - Affected modules and functionalities:
     conda build
     tests
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     NA

**JIRA TASK**: *[NA]*

### CODE REVIEW

File: qa/TL1_python-self-test_conda/test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+./test_nofw.sh
+./test_cupy.sh
```
Comment (jantonguirao) [inline qa/TL1_python-self-test_conda/test.sh]: missing empty line

Comment (JanuszL) [reply qa/TL1_python-self-test_conda/test.sh]: Done

File: conda/recipe/build.sh [hunk 2]
```diff
@@ -158,7 +158,16 @@ export LD_LIBRARY_PATH="$PREFIX/libjpeg-turbo/lib:$PREFIX/lib:$LD_LIBRARY_PATH"
 DALI_PATH=$($PYTHON -c 'import nvidia.dali as dali; import os; print(os.path.dirname(dali.__file__))')
 echo "DALI_PATH is ${DALI_PATH}"
 pushd $SRC_DIR/dali_tf_plugin/
-source ./build_dali_tf.sh $DALI_PATH/plugin/libdali_tf_current.so
+mkdir -p prebuilt/plugin
+source ./build_dali_tf.sh prebuilt/plugin/libdali_tf_current.so
```
Comment (jantonguirao) [inline conda/recipe/build.sh:161]: I think you can skip that, as it is going to be build during `pip install`

Comment (JanuszL) [reply conda/recipe/build.sh]: Done

**Pull request merged**